src/commentbot/parse_comment.jl

```diff
@@ -0,0 +1,26 @@
+const accept_regex = "([^\\r\\n]*)(\\n|\\r)*.*"
+
+function parse_comment(fncall)
+    argind = findfirst(isequal('('), fncall)
+    name = fncall[1:(argind - 1)]
+    parsed_args = Meta.parse(replace(fncall[argind:end], ";" => ","))
```

Comment:
This smells like a security hole

Reply:
This code long since in production, so this isn't really the place to comment on it (although it's hard to tell what is and isn't new with all the file renaming). Relevant: #130